From 2b95f71b9830e58003b52dff1a6be01390096c5a Mon Sep 17 00:00:00 2001
From: Tim Rakowski <tim.rakowski@googlemail.com>
Date: Mon, 8 Jan 2018 22:19:58 +0100
Subject: [PATCH] setup.py: Exclude all tests* from install

- Fixes issue #498
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 3188a94..b463d6b 100644
--- a/setup.py
+++ b/setup.py
@@ -110,7 +110,7 @@ def run(self):
 setup(
     name='catkin_tools',
     version='0.4.4',
-    packages=find_packages(exclude=['tests', 'docs']),
+    packages=find_packages(exclude=['tests*', 'docs']),
     package_data={
         'catkin_tools': [
             'notifications/resources/linux/catkin_icon.png',

